specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-07-01/examples/Compute/get/AmlCompute.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "computeName": "compute123",
+    "api-version": "2021-07-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
>body [](http://example.com/codeflow?start=7&length=4)

The GET response here is very different than in the `createOrUpdate` example responses. Is that just an oversight or intentional?

Reply:
I think it's oversight but because it's owned by a partner team, I can't speak for them. But since this example is inherited from previous versions, do we have to change it in this PR?

Reply:
fixed the example